modules/sre-cluster-access.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa_policy_service_definition/rosa-sre-access.adoc
+
+:_content-type: CONCEPT
+[id="sre-cluster-access_{context}"]
+= SRE cluster access
+
+SRE access to {product-title} (ROSA) clusters can get full cluster administrative access through an escalation procedure; this procedure consists of several layers of authentication and multiple streams of approval, all of which are managed by strict company policy. All authentication attempts to access a cluster, as well as changes made within a ROSA cluster, are recorded within audit logs, along with the specific account identity of the SRE responsible for those actions. These audit logs help ensure that all changes made by SREs to a customer's ROSA cluster adhere to the strict policies and procedures that make up Red Hat's managed services guidelines.
```

Comment:
Nit: I found this compound sentence a bit hard to read.

"SRE access to {product-title} (ROSA) clusters can get full cluster administrative access through an escalation procedure; this procedure consists of several layers of authentication and multiple streams of approval, all of which are managed by strict company policy." 

I think just changing the semicolon to a period would be sufficient, but you may want a second opinion. It just reads long.